packages/swc-configs/src/cloneObjectExceptFunctions.ts

```diff
@@ -0,0 +1,3 @@
+export function cloneObjectExceptFunctions(obj: unknown) {
```

Comment:
The return type is the same as the input type

export function cloneObjectExceptFunctions<T>(obj: T): T